penguin/tools/browser_harness_tools.py

```
@@ -0,0 +1,870 @@
+            info = helpers.page_info()
+        return {
+            "result": "Switched browser tab",
+            "target_id": target_id,
+            "session_id": session_id,
+            "page_info": info,
+            "backend": "browser-harness",
+        }
+
+
+class BrowserHarnessStatusTool:
+    def __init__(self, config: Optional[dict[str, Any]] = None):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)